tools/buildFactory.ts

```diff
@@ -348,27 +377,56 @@ export function gulpIsolateFactory(
     });
 
     await jsRollupBundle.write({
-      sourcemap: true,
-      file: `${output}/index.js`,
-      format,
-      exports: 'named'
+      sourcemap: initializedSettings.sourceMap,
+      file: `${output}/${initializedSettings.outputFileName}`,
+      format: initializedSettings.format,
+      exports: 'named',
+      name: 'sdsl'
     });
 
     await jsRollupBundle.close();
   }
 
-  build.displayName = `${format}-build`;
+  build.displayName = `${initializedSettings.format}-build`;
 
   function mangle() {
     return gulp
-      .src([`${output}/index.js`])
-      .pipe(sourcemaps.init({ loadMaps: true }))
+      .src([`${output}/${initializedSettings.outputFileName}`])
+      .pipe(initializedSettings.sourceMap
+        ? sourcemaps.init({ loadMaps: true })
+        : tap(() => { /* */ }))
       .pipe(terserStream())
-      .pipe(sourcemaps.write('.'))
+      .pipe(initializedSettings.sourceMap ? sourcemaps.write('.') : tap(() => { /* */ }))
       .pipe(gulp.dest(`${output}`));
   }
 
-  mangle.displayName = `${format}-mangle`;
+  mangle.displayName = `${initializedSettings.format}-mangle`;
+
+  function generateMin() {
+    let minFileName = path.basename(
```

Comment:
This seems not work.

All the name is `js-sdsl.min.js` but expect to `js-sdsl-${module}.min.js` or `js-sdsl.${module}.min.js`.

Reply:
I don't understand what does this variable do.

Reply:
The `minFileName` variable has a value that converts the "js-sdsl.js" to "js-sdsl.min.js".

Reply:
So why not assign it directly?

`minFileName = js-sdsl.ming.js`

Reply:
Receiving "js-sdsl.js" outside of the function.

because this function needs to operate in a general way.